Formula/vulkan-loader.rb

```diff
@@ -0,0 +1,35 @@
+class VulkanLoader < Formula
+  desc "Vulkan ICD Loader"
+  homepage "https://github.com/KhronosGroup/Vulkan-Loader"
+  url "https://github.com/KhronosGroup/Vulkan-Loader/archive/refs/tags/v1.3.220.tar.gz"
+  sha256 "fe5b65e5d88febb1220ae59de363ed8b2794f8c861d6d74efb14aecefd464559"
+  license "Apache-2.0"
+  head "https://github.com/KhronosGroup/Vulkan-Loader.git", branch: "master"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.10" => :build
+  depends_on "vulkan-headers"
+
+  def install
+    system "cmake", ".", "-DVULKAN_HEADERS_INSTALL_DIR=#{Formula["vulkan-headers"].opt_prefix}",
+           "-DFALLBACK_DATA_DIRS=#{HOMEBREW_PREFIX}/share:/usr/local/share:/usr/share",
```

Comment:
What is `FALLBACK_DATA_DIRS` used for?

Reply:
Shouldn't we have
```ruby
Formula["vulkan-headers"].opt_share
```
in `FALLBACK_DATA_DIRS` then? Or is that made redundant by `VULKAN_HEADERS_INSTALL_DIR`?

Reply:
We could, but it'd be pretty unusual (and obviously inappropriate on Linux). Generally, the loader is expected to look in a list of standard dirs provided by the OS (or package manager) for _all_ drivers, and each driver is expected to install a .json file in one of those dirs; the loader isn't meant to have to know anything about each individual driver at compile-time. This mechanism also allows other packages to link against `vulkan-loader` without creating a heavier hard dependency on a driver, and lets the user enable or disable a particular driver at runtime by just linking or unlinking it.

The fact that there's only one Vulkan driver in Brew right now is sorta just a coincidence; Linuxbrew could expose several more from the `mesa` formula, there are at least two others for AMD GPUs (plus one for Nvidia), and it's entirely plausible that we could one day see multiple implementations on macOS (like how ANGLE and MoltenGL coexist).

We see similar behavior in the formulae for `pkg-config`, `automake`, `fish`, `zsh`, and a number of others that want to allow other packages to provide resources for them to use at runtime.